packages/react/src/Footer/MenuLink.client.tsx

```diff
@@ -29,7 +32,7 @@ export const MenuLink = ({ links, isAboutOpen = false }: MenuLinkProps) => {
           href={menuItem.link}
           target={menuItem.openInCurrentTab ? "_top" : "_blank"}
           rel="noreferrer"
-          {...(!isAboutOpen && { tabIndex: -1 })}
+          {...(isSmallScreen && !isAboutOpen && { tabIndex: -1 })}
```

Comment:
ca fait mal à la tete cette syntaxe. je pense qu'etre plus explicite ca serait mieux 
```suggestion
          tabIndex={(isSmallScreen && !isAboutOpen) ? -1: null }
```